templates/barbican/config/00-default.conf

```diff
@@ -44,15 +44,38 @@ enable = true
 
 [secretstore]
 enable_multiple_secret_stores = true
-stores_lookup_suffix = simple_crypto
+stores_lookup_suffix = {{ .enabledSecretStores }}
```

Comment:
You need to add logic to pass this parameter in -- string.join(instance.Spec.EnabledSecretStores, ",")

Reply:
Same for the *CryptoEnabled .. etc. as in our google doc.